insights/parsers/satellite_postgresql_query.py

```diff
@@ -12,6 +12,8 @@
 -------------------------------------------------------------------------------------------------------------------------------
 SatelliteKatellloReposWithMultipleRef - command ``psql -d foreman -c "select repository_href, count(*) from katello_repository_references group by repository_href having count(*) > 1;" --csv``
 ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+SatelliteLogsTableSize - command ``psql -d foreman -c "select pg_size_pretty(pg_total_relation_size('logs')) as logs_size" --csv``
+----------------------------------------------------------------------------------------------------------------------------------
```

Comment:
As discussed, better removed the "pg_size_pretty" function from the SQL and also remove the calculation from the parser.

Reply:
@xiangce updated as suggested, please help to review.